src/token-pool.js

```diff
@@ -83,9 +89,6 @@ class TokenPool {
       });
       console.warn(`Adding ${res.length} tokens to acquired pool`);
       this.tokens.push(...res);
-    } else if (response.status === 401){
```

Comment:
A 401 from the token endpoint means that the access token is expired. Don't we also need to handle this case by forcing a refresh? Cookies are not always deleted immediately on expiry, so we could exhaust our tokens before the cookie listener is called. It would be possible to trigger it by manually deleting the cookie with the `cookie` API.

Reply:
Ah yeah, added this back